tests/test_interp_utils.py

```diff
@@ -396,6 +421,99 @@ def test_fourier_chebyshev(self, func, m, n):
         np.testing.assert_allclose(fc.evaluate(m, n), f)
 
 
+class TestStreams:
+    """Test convergence of inverse stream maps."""
+
+    tol = 1e-8
+    norm = LogNorm(1e-7, 1e0)
+    X = 48
+    Y = 48
+    rho = 0.6
+
+    @staticmethod
+    def theta_chebyshev(eq, X, Y, rho, tol):
+        """Chebyshev spectrum of θ(α, ζ)."""
+        zeta = cheb_pts(Y, (0, 2 * np.pi / eq.NFP))[::-1]
+        Λ = get_transforms(
+            "lambda", eq, LinearGrid(rho=rho, M=eq.L_basis.M, zeta=zeta, NFP=eq.NFP)
+        )["L"]
+
+        theta = eq._map_poloidal_coordinates(
+            jnp.atleast_1d(eq._compute_iota_under_jit(rho)),
+            cheb_pts(X, (0, 2 * np.pi)),
+            zeta,
+            eq.params_dict["L_lmn"],
+            Λ,
+            inbasis=("rho", "alpha", "zeta"),
+            outbasis=("rho", "theta", "zeta"),
+            tol=tol,
+        ).squeeze(0)[..., ::-1]
+
+        c = ChebyshevSeries(theta, (0, 2 * np.pi), (0, 2 * np.pi / eq.NFP))._c
+        c = cheb_from_dct(cheb_from_dct(c, -1), -2)
+        return np.abs(c)
+
+    @pytest.mark.unit
+    @pytest.mark.parametrize("name", ["W7-X", "NCSX", "HELIOTRON"])
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
+    @staticmethod
+    def test_theta_chebyshev(name):
+        """Plot Chebyshev spectrum of θ(α, ζ)."""
+        eq = get(name)
+        X = TestStreams.X
+        Y = TestStreams.Y
+        rho = TestStreams.rho
+        c = TestStreams.theta_chebyshev(eq, X, Y, rho, tol=TestStreams.tol)
+
+        fig, ax = plt.subplots()
+        ax.set(ylabel=r"$x$", xlabel=r"$y$")
+        ax.set_title(
+            r"Projection of "
+            r"$\alpha, \zeta \mapsto \theta$ onto "
+            r"$\{T_x(\alpha / \pi - 1) T_y(N_{\text{FP}} \zeta / \pi - 1)\}$"
+            r"$_{\text{Chebyshev}}$",
+            pad=20,
+        )
+        plt.matshow(c, fignum=0, norm=TestStreams.norm, cmap="turbo")
+        cbar = plt.colorbar(orientation="horizontal")
+        cbar.ax.invert_xaxis()
+        return fig
+
+    @pytest.mark.unit
+    @pytest.mark.parametrize("name", ["W7-X", "NCSX", "HELIOTRON"])
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
+    @staticmethod
+    def test_delta_fourier_chebyshev(name):
+        """Plot Fourier-Chebyshev spectrum of δ(α, ζ)."""
+        eq = get(name)
+        X = TestStreams.X
+        Y = TestStreams.Y
+        angle = Bounce2D.angle(eq, X, Y, TestStreams.rho, tol=TestStreams.tol)
+        return Bounce2D.plot_angle_spectrum(angle, 0, norm=TestStreams.norm)
+
+    @pytest.mark.unit
+    @pytest.mark.parametrize("name", ["W7-X", "NCSX", "HELIOTRON"])
+    @pytest.mark.mpl_image_compare(remove_text=True, tolerance=tol_2d)
+    @staticmethod
+    def test_lambda_fourier_vartheta_zeta(name):
+        """Plot Fourier spectrum of Λ(ϑ, ζ)."""
```

Comment:
is this lowercase lambda? (ie our usual poloidal stream function?)

Reply:
yes Λ is DESC stream. little lambda is pitch angle